api/models/recipesModel.js

```diff
@@ -0,0 +1,53 @@
+const db = require('../../data/dbConfig.js')
+
+module.exports = {
+    find,
+    findById,
+    findIngredientsById,
+    findDishesById,
+    add,
+    update,
+    remove
+}
+
+function find() {
+    return db('recipes')
+}
+
+function findById(id) {
+    return db('recipes')
+        .where({ id })

```

Comment:
I would attach the ingredients here as well. I know it wasn't stated in MVP, but I can't think of a reason why you would get a recipe by id and not need the ingredients with it